pretixprint/app/src/main/java/eu/pretix/pretixprint/print/ESCPOSRenderer.kt

```diff
@@ -516,6 +520,31 @@ class ESCPOSRenderer(private val dialect: Dialect, private val receipt: JSONObje
     }
 
     private fun mode(font: String = "a", emph: Boolean = false, doubleheight: Boolean = false, doublewidth: Boolean = false, underline: Boolean = false) {
+        if (dialect == Dialect.StarPRNT) {
+            // font
+            out.add(ESC)
+            out.add(RS)
+            out.add('F'.toByte())
+            when(font) {
+                "a" -> out.add(0)
+                "b" -> out.add(1)
+                "c" -> out.add(2)
```

Comment:
```suggestion
                FONTA -> out.add(0)
                FONTB -> out.add(1)
                FONTC -> out.add(2)
```

And add `FONTC` to the companion object